panels/nerf_panel_operators/import_dataset_operator.py

```diff
@@ -45,13 +50,18 @@ class ImportNeRFDatasetOperator(bpy.types.Operator):
 
     @classmethod
     def poll(cls, context):
-        return NeRFManager.can_import()
+        return NeRFManager.can_import
```

Comment:
as far as i can tell, this is still a function (not a property).  It may evaluate to be truthy here, but possible because the `NeRFManager.can_import` method is not `None` - until you overwrite it with a `True` or `False` value

Reply:
After import i set can import to true and after removal i set it to false